modules/tictactoe.py

```diff
@@ -0,0 +1,75 @@
+from .base import Module
+
+
+class TicTacToe(Module):
+    DESCRIPTION = "Play a friendly game of tic tac toe"
+    ARGC = 1
+    players = [''] * 2
+    board = [' '] * 9
```

Comment:
Hm... I feel like maybe it would be better to use an enum than a string and use a 2D array.